packages/typespec-test/test/openai_modular/generated/typespec-ts/src/models/models.ts

```diff
@@ -58,7 +58,7 @@ export function audioTranscriptionOptionsSerializer(
       : [{ name: "filename", body: item["filename"] }]),
     ...(item["responseFormat"] === undefined
       ? []
-      : [{ name: "response_format", body: item["responseFormat"] }]),
+      : [{ name: "responseFormat", body: item["responseFormat"] }]),
```

Comment:
This is expected and should be a bugfix in our side. The property named `response_format` is defined as JSON wire name not the multipart name. So we should not adopt this name.